tests/vexflow_test_helpers.ts

```diff
@@ -115,7 +115,17 @@ const SVG_TEST_CONFIG = {
   backend: Renderer.Backends.SVG,
   tagName: 'div',
   testType: 'SVG',
-  fontStacks: ['Bravura', 'Gonville', 'Petaluma', 'Leland', 'MuseJazz'],
+  fontStacks: [
+    'Bravura',
+    'FinaleAsh',
+    'FinaleBroadway',
+    'FinaleMaestro',
```

Comment:
If the official font names have a space, why don't we be consistent and keep the space? Otherwise it'll get confusing later (remember with Roboto Slab and Petaluma Script)?

Reply:
Done